raft/src/main/java/org/apache/kafka/raft/RaftConfig.java

```diff
@@ -59,6 +60,11 @@
     private static final String QUORUM_LINGER_MS_DOC = "The duration in milliseconds that the leader will " +
         "wait for writes to accumulate before flushing them to disk.";
 
+    public static final String QUORUM_FLUSH_MIN_SIZE_BYTES_CONFIG = QUORUM_PREFIX + "flush.minSize.bytes";
```

Comment:
If we decide to keep this, how about we call it `quorum.append.max.unflushed.bytes` or something like that? Basically it's the maximum number of bytes that the raft implementation is allowed to accumulate before forcing an fsync.

Reply:
I like the suggestion of using `quorum.append.max.unflushed.bytes`. I would mention in the documentation that this is a best effort. Based on how `KafakRaftClient` calls drain, it is possible for accumulator have more unflushed bytes.